src/bluehawk/parser/visitor/makeCstVisitor.ts

```diff
@@ -332,8 +355,17 @@ export function makeCstVisitor(
           errors,
           source,
         });
-        this.visit(context.chunk, { parent: newNode, errors, source });
 
+        if (context.blockCommandUncommentedContents != undefined) {
```

Comment:
This entire if/else could be rewritten as:

```js
this.visit(context.blockCommandUncommentedContents  ?? context.chunk, { parent: newNode, errors, source });
```

Reply:
Thanks for the note! I agree that seems cleaner